test/UnitTest/Microsoft.AspNet.OData.Test/PublicApi/Microsoft.AspNet.OData.PublicApi.bsl

```diff
@@ -2354,9 +2354,48 @@ public enum Microsoft.AspNet.OData.Query.SelectExpandType : int {
 	Disabled = 2
 }
 
+<<<<<<< HEAD
```

Comment:
Fix merge result

Reply:
working on it, currently dealing with an issue where VS 2019 16.8 will not run unit tests on local machine. see tail end of microsoft/vstest#1220